testing/README.md

```diff
@@ -0,0 +1,72 @@
+## Testing
+
+This directory is used for multi-kernel testing using litte-vm-helper
+(https://github.com/cilium/little-vm-helper/).
+
+### Usage
+
+NOTE: For now, commands need to be executed from the top-level tetragon directory.
+
+
+Build everything:
+
+```
+$ make -C testing
+make: Entering directory '/home/kkourt/src/tetragon/testing'
+go build ../cmd/tetragon-tester
+go build ../cmd/tetragon-vmtests-run
+make: Leaving directory '/home/kkourt/src/tetragon/testing'
+```
+
+
+Download rootfs images and kernels in `testing/test-data`
+```
+$ ./testing/fetch-data.sh 5.4  bpf-next
+...
+$ ls testing/test-data/images testing/test-data/kernels
+testing/test-data/images:
+base.qcow2
+
+testing/test-data/kernels:
+5.4/  bpf-next/
+```
+
+Run tests on 5.4:
+```
+$ make test-compile # <- this will build the test binaries
+$ ./testing/tetragon-vmtests-run \
+	--kernel testing/test-data/kernels/5.4/boot/vmlinuz-5.4.206 \
+	--base testing/test-data/images/base.qcow2
+```
+
+Run tests on 5.4, without KVM acceleration. Doing so, makes the run closer to the GH action
+environment (which do not support nested virtualization) and it  also uncover other issues due to
+the unconventional timing.
+
+```
+$ ./testing/tetragon-vmtests-run \
+	--kernel testing/test-data/kernels/5.4/boot/vmlinuz-5.4.206 \
+	--base testing/test-data/images/base.qcow2 \
+	--qemu-disable-kvm
+```
```

Comment:
I'm getting fail on fedora:
```
$ ./testing/tetragon-vmtests-run --kernel testing/test-data/kernels/5.4/boot/vmlinuz-5.4.206 --base testing/test-data/images/base.qcow2 --qemu-disable-kvm
Error: required cmd 'mmdebstrap' not found
```
mmdebstrap seems to be debian specific tool, is this limited to debian or perhaps ubuntu?

Reply:
It is specific to debian based distros. One workaround could be to package a Docker container with mmdebstrap installed.

Reply:
Ah good catch, thanks @olsajiri! `mmdebstrap` is not needed here, so I'll just remove the check in:
https://github.com/cilium/little-vm-helper/blob/9e7539fbeba9ef971f98c33e84282db7de3743ac/pkg/images/env.go#L16-L21

Reply:
@olsajiri can you try again with latest? I believe everything should work now.